tests/utils/settings.ts

```diff
@@ -0,0 +1,26 @@
+export interface Settings {
```

Comment:
You don't import this file, so do you actually need it?

Reply:
done